### PR TITLE
Add scripts to any post edit page

### PR DESCRIPTION
**Pull request by balbuf**
Looks like some whitespace updates snuck in there too

### CODE REVIEW

**Comment by balbuf** [comment]
(`$current_screen->id` on a post edit page === post type slug)

**Comment by vhuynh617** [comment]
Amazing 🙆 /approved

File: wp-remember-the-galleries/wp-remember-the-galleries.php [hunk 1]
```diff
@@ -248,7 +248,7 @@ static function enqueue_scripts() {
 			)
 		);
 
-		if( $current_screen->id === 'upload' || $current_screen->id === 'post' ||
+		if( $current_screen->id === 'upload' || $pagenow === 'post.php' ||
```
Comment (bendoh) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: So what's the difference here? I Swear I wrote this! Is it 'post' vs 'edit' ?

Comment (balbuf) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: `post.php` is the general page for editing any kind of post...`$current_screen->id` === post type slug. I inspected `$current_screen` and didn't see anything that would definitively show we were on any post edit page, so I think `$pagenow` is better for this. Also you did write this 😼#thxgitblame

Comment (balbuf) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: and `edit.php` is the page which lists posts for editing I believe. Now that I'm thinking about it though, I think on a new post the `$pagenow` is going to be `post-new.php`. Lemme confirm this before merging.

Comment (bendoh) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: What about `$current_screen->id === get_post_type()` ?

Comment (balbuf) [inline wp-remember-the-galleries/wp-remember-the-galleries.php]: Seems like that would probably work too...gotta check both `post.php` and `post-new.php` variants

**Comment by balbuf** [comment]
Went with your suggestion @bendoh -- works on `post.php` and `post-new.php`. I hope WP doesn't use this as the ID for any other pages, but it wouldn't be catastrophic anyway. I checked `edit.php` and the id is `edit-{post_type_slug}`.

Anyway, please re-review!

**Pull request merged**